src/audio/volume/volume.c

```diff
@@ -587,7 +587,7 @@ static int volume_process(struct processing_module *mod,
 		cd->peak_cnt = 0;
 		peak_vol_update(cd);
 		memset(cd->peak_regs.peak_meter, 0, sizeof(cd->peak_regs.peak_meter));
-#ifdef VOLUME_HIFI4
+#if CONFIG_VOLUME_HIFI4
```

Comment:
why not this?
```suggestion
#if SOF_USE_HIFI(4, VOLUME)
```

Reply:
Do not resolve comments without a single word or single change
https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#workflow-suggestions-that-help-reviewers

Reply:
yes, just too much comments and I don't want to use a new PR to submit this patch, let's go for merge.